packages/matchbox/src/components/Button/styles.js

```diff
@@ -1,7 +1,50 @@
 import { tokens } from '@sparkpost/design-tokens';
 
+export function focus(props) {
+  let color = tokens.color_gray_300;
+
+  switch (props.buttonColor) {
+    case 'orange': // To be deprecated
+    case 'blue':
+      color = tokens.color_blue_700;
+      break;
+
+    case 'red':
+      color = tokens.color_red_700;
+      break;
+
+    case 'gray':
+    default:
+      color = tokens.color_gray_900;
+      break;
+  }
+
+  return `
+    position: relative;
+    outline: none;
+
+    &:after {
+      position: absolute;
+      content: "";
+      display: block;
+      top: -3px;
+      left: -3px;
+      right: -3px;
+      bottom: -3px;
+      transition: ${tokens.motionDuration_fast};
+      border-radius: ${tokens.borderRadius_200};
+      box-shadow: none;
+      pointer-events: none;
+    } 
+
+    &:focus:after {
+      z-index: 1;
+      opacity: 1;
+      box-shadow: 0 0 0 2px ${color};
```

Comment:
add this to box-shadow tokens?

Reply:
Hmm can't really make this a token since it relies on the other code in here. What i can do though is make this a reusable helper so we can use it in other places